test/checkpoint/workers.c

```diff
@@ -392,6 +392,8 @@ real_worker(void)
         }
 
     for (i = 0; i < g.nops && g.running; ++i, __wt_yield()) {
+        if (i > 0 && i % 5000 == 0)
+            printf("Worker %u of %u ops\n", i, g.nops);
```

Comment:
No objection to the progress `printf`.  Just want to confirm that it's not debug code that accidentally got included in the PR.

Reply:
Yes, it was intentional.